bin/run_cluster_tests_in_ci.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -ex
+
+lein clean
+mv -fv resources/config.test.{cluster.ci.edn,cluster.edn}
```

Comment:
Why are there two config files for ci and others?

Reply:
The config is exactly the same, but that's how it has been for tests and CI.
Not aware of the rationale behind keeping two files.
But, we can remove them and keep just one file for each docker-compose.

Reply:
actually, we should have just one `config.test.edn` file. We don't even need `config.test.ci.edn` or other variants (datadog, statsd, cluster, etc, etc). These are not serving ANY purpose.

WDY all think?

Reply:
We can take this task up as a separate issue.